package.json

```diff
@@ -30,44 +30,44 @@
     ]
   },
   "dependencies": {
-    "@babel/parser": "^7.9.4",
-    "@babel/runtime": "^7.9.2",
-    "chalk": "^3.0.0",
-    "escodegen": "^1.14.1",
-    "fs-extra": "^8.1.0",
-    "globby": "^11.0.0",
-    "into-stream": "^5.1.1",
+    "@babel/parser": "^7.13.10",
+    "@babel/runtime": "^7.13.10",
+    "chalk": "^4.1.0",
+    "escodegen": "^2.0.0",
+    "fs-extra": "^9.1.0",
+    "globby": "^11.0.2",
+    "into-stream": "^6.0.0",
     "minimist": "^1.2.5",
-    "multistream": "^2.1.1",
+    "multistream": "^4.1.0",
     "pkg-fetch": "^2.6.9",
     "progress": "^2.0.3",
-    "resolve": "^1.15.1",
+    "resolve": "^1.20.0",
     "stream-meter": "^1.0.4"
   },
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.9.0",
-    "@babel/plugin-transform-async-to-generator": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.9.0",
-    "@babel/preset-env": "^7.9.0",
-    "@babel/register": "^7.9.0",
+    "@babel/cli": "^7.13.10",
```

Comment:
I'm not sure it's worth the risk to update all dependencies at the same time as adding a new feature. Thoughts?

Reply:
All tests passes .... do we have enough coverage to gain confidence ?

Reply:
I have move this chagne  to a dedicated pull request